configs/template_config.yaml

```diff
@@ -81,6 +81,13 @@ tokenized_dataset_path: "<YOUR_PATH_HERE>/data/tokenized_dataset/c4"
 # TensorBoard Path (str): Path for where TensorBoard files go, optional
 tboard_path: "<YOUR_PATH_HERE>/data/runs"
 
+# This is where you can specify what percentage of C4 data you want to use for training
```

Comment:
Please make this code work with datasets beyond C4 as well.

Reply:
Still agree with Kimball on this one. This could be as simple as changing the comment, right @calvin-seamons?